src/main/java/group1/intern/controller/ProfilesController.java

```diff
@@ -0,0 +1,148 @@
+package group1.intern.controller;
+
+import group1.intern.annotation.CurrentAccount;
+import group1.intern.bean.PasswordInfo;
+import group1.intern.bean.ProfileInfo;
+import group1.intern.bean.ToastMessage;
+import group1.intern.model.Account;
+import group1.intern.model.Enum.AccountRole;
+import group1.intern.service.AuthService;
+import group1.intern.util.exception.BadRequestException;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
+import org.springframework.web.servlet.mvc.support.RedirectAttributes;
+
+@RequestMapping("profiles")
+@Controller
+@RequiredArgsConstructor
+public class ProfilesController {
+    @Autowired
+    private final AuthService authService;
+
+    @GetMapping
+    public String profiles(Model model, @CurrentAccount Account currentAccount) {
+        model.addAttribute("profileInfo", ProfileInfo.builder()
+            .accountId(currentAccount.getId())
+            .fullName(currentAccount.getFullName())
+            .dateOfBirth(currentAccount.getDateOfBirth())
+            .phoneNumber(currentAccount.getPhoneNumber())
+            .email(currentAccount.getEmail())
+            .gender(currentAccount.getGender())
+            .address(currentAccount.getAddress())
+            .avatarUrl(currentAccount.getAvatarUrl())
+            .displayName(currentAccount.getDisplayName())
+            .build());
+        model.addAttribute("currentAccount", currentAccount);
+        model.addAttribute("activeTab", "home");
+        model.addAttribute("passwordInfo", new PasswordInfo(currentAccount.getId(), currentAccount.getPassword(), currentAccount.getPassword()));
+        if (currentAccount.getRole() == AccountRole.CUSTOMER) {
+            return "screens/customer/profiles/show";
+        } else if (currentAccount.getRole() == AccountRole.SELLER) {
+            return "screens/seller/profiles/show";
+        }
+        return "redirect:/";
+    }
+
+    @PostMapping("/change-profile")
```

Comment:
```suggestion
    @PostMapping("/")
```
hoặc bỏ luôn`/` em nghĩ sao